apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -419,21 +419,21 @@
                                 }
                                 if (localAuthenticatorNames.contains(MAGIC_LINK_AUTHENTICATOR)) {
                             %>
-                            <div class="field">
-                                <button class="ui grey labeled icon button fluid"
-                                    onclick="handleNoDomain(this,
+                            <div class="social-login blurring social-dimmer">
+                                <div class="field">
+                                    <button class="ui button" onclick="handleNoDomain(this,
                                         '<%=Encode.forJavaScriptAttribute(Encode.forUriComponent(idpEntry.getKey()))%>',
-                                        '<%=MAGIC_LINK_AUTHENTICATOR%>')"
-                                    id="icon-<%=iconId%>"
-                                    title="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%>
-                                    <%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with" )%>">
-                                    <i class="email icon"></i>
-                                    <img src="libs/themes/default/assets/images/icons/magic-link-icon.svg" alt="Magic Link Logo" />
-                                    <span>
-                                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with" )%>
-                                        <%=AuthenticationEndpointUtil.i18n(resourceBundle, "magic.link" )%>
-                                    </span>
-                                </button>
+                                        '<%=MAGIC_LINK_AUTHENTICATOR%>')" id="icon-<%=iconId%>"
+                                        title="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with")%>
+                                            <%=AuthenticationEndpointUtil.i18n(resourceBundle, "magic.link" )%>"
+                                        data-componentid="login-page-sign-in-with-magic-link">
+                                        <img class="ui image" src="libs/themes/default/assets/images/icons/magic-link-icon.svg" alt="Magic Link Logo" />
+                                        <span>
+                                            <%=AuthenticationEndpointUtil.i18n(resourceBundle, "sign.in.with" )%>
+                                            <%=AuthenticationEndpointUtil.i18n(resourceBundle, "magic.link" )%>
```

Comment:
Let's remove the white spaces before the closing parenthesis.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3586